code_of_conduct.md

```diff
@@ -0,0 +1,253 @@
+- The fact-finding team may utilize whatever reasonable investigative methods are at their disposal to investigate whether an offense has been committed. These methods include, but are not limited to:
+  - In-person or telephone interviews of witnesses or the accused
+  - Written Statements
+  - Requests for written information
+  - On-site visits
+  - Fact-finding conferences
+- The fact-finding team must present its findings within ten (10) days of the conclusion of its investigation. If the fact-finding team finds that the allegations are unfounded, it must present its report to the Standards Board and VP of Standards. If the allegations are determined to have merit, the chairman of the fact-finding team should prepare the matter to be presented to the Standards Board, VP of Standards.
+
+**Section 4.** Rights of the Accused
+
+- Any Member charged with an offense shall have the right to a hearing before the Standards Board and VP of Standards.
+- The Accused shall have the right of timely notice of hearing, as hereinafter provided.
+- The Accused has the right to be heard during the time disciplinary action is being considered.
+- The Accused shall have the right to see the anonymous written charges prior to any hearing.
+- The Accused shall have the right to select his own counsel from the membership of the Fraternity, but it is emphasized that the disciplinary hearing procedures are those of Texas Lambda Alpha Nu and are not the procedures that would be utilized in a court of law.
+- Counsel or the Member may be asked to leave the hearing at any time by the presiding officers if, in the sole judgment of the presiding officers, counsel or the Member is disrupting the hearing.
+
+**Section 5.** Hearings
+
+- If the Standards Board and VP of Standards elect to treat the matter as a formal complaint, then all charges shall be presented to the accused Member in written form. A time shall be set for a hearing, no more than 15 business days after the Member has been notified. Maximum time limits for scheduling of hearings may be extended at the discretion of the Standards Board and VP of Standards.
+- At the discretion of the Standards Board and VP of Standards, instead of a face-to-face hearing, participants in a hearing may be allowed to participate telephonically or submit written information.
+- The complainant may choose to remain anonymous to the accused and the Standards Board and VP of Standards shall respect the complainant&#39;s wish.
+- The Standards Board and VP of Standards may conduct a formal hearing according to the following guidelines:
+  - The presiding officer, the VP of Standards, shall call the hearing to order.
+  - Admission of any person to the hearing shall be at the discretion of the presiding officer.
```

Comment:
Kinda falling back to my previous comment, the idea behind having the hearings in private is to prevent unnecessary speculation from the general membership. I'm open to discussion about this though.